_posts/2015-03-03-codeofconduct.md

```diff
@@ -61,3 +61,9 @@ We will respect confidentiality requests for the purpose of protecting victims o
 Participants asked to stop any harassing behavior are expected to comply immediately.
 
 If a participant engages in harassing behavior, the administrators may take any action they deem appropriate, up to and including expulsion from all EMS spaces and identification of the participant as a harasser to other EMS members or the general public.
+
+## Advertising
+EMS is run by a team of dedicated admins for the enjoyment and benefit of it's members. Our Admins give up their own time to foster and develop the community and as such we aim to keep the community an advertising free zone. We take a firm stance on members posting advertisements of any kind in the community and we reserve the right to remove posts which fall foul of this rule.
```

Comment:
minor nits:
- it's -> its
- Admins -> admins
- advertising free -> advertising-free
- foul -> afoul